content/03.methods.md

```diff
@@ -48,6 +48,14 @@ We trained this model using scikit-learn's [@raw:scikit-learn] implementation of
 For each principal component we calculated its cosine similarity with  all tokens in our word2vec model's vocabulary.
 We report the top 100 positive and negative scoring tokens in the form of  word clouds, where the size of each word corresponds to the magnitude of similarity and color represents positive (blue) or negative (orange) association.
 
-### Recommending Journals/ bioRxiv Audience Analysis
-1. This title will update as analysis is completed
-2. This section will describe how the above process is conducted
+### Journal Recommendation
+We trained two k-nearest neighbor classifiers to recommend journals based on article content.
+The first classifier recommends journals based on articles in close proximity to the query and the second classifier recommends journals based on close proximity to each journal's centroid.
+We used PubMed Central's (PMC) open access dataset to train both models.
+We preprocessed this dataset by reserving articles that were found in both PMC and bioRxiv for our test set and used the rest for training.
+Furhermore, we filtered out journals that had less than 100 articles within PMC and downsampled the remaining journals to only have 100 articles as this prevents overrepresentation of mega journals.
```

Comment:
```suggestion
Furhermore, we filtered out journals that had fewer than 100 articles within PMC and downsampled the remaining journals to only have 100 articles, which prevented mega journals from completely dominating the predictions.
```

Reply:
Did you do this just for the article-based predictor? I thought you were going to calculate centroids with all papers.